### PR TITLE
The forms.rst file has been translated into Portuguese.

### PR DESCRIPTION
**Pull request by MatheusRibak**

### CODE REVIEW

File: pt/index.rst [hunk 1]
```diff
@@ -13,7 +13,7 @@ Phalcon não é somente um framework de alta performance, nosso objetivo é faze
 Tabela do Conteúdo
 ------------------
 
-Installation
+Instalação
 ^^^^^^^^^^^^
```
Comment (sergeyklay) [inline pt/index.rst]:
```diff
--- a/pt/index.rst
+++ b/pt/index.rst
 Instalação
-^^^^^^^^^^^^
+^^^^^^^^^^

File: pt/index.rst [hunk 2]
```diff
@@ -43,7 +43,7 @@ Tutorials
    reference/tutorial-rest
    reference/samples
 
-Components
+Componentes
 ^^^^^^^^^^
```
Comment (sergeyklay) [inline pt/index.rst]:
```diff
--- a/pt/index.rst
+++ b/pt/index.rst
 Componentes
-^^^^^^^^^^
+^^^^^^^^^^^

File: pt/index.rst [hunk 3]
```diff
@@ -102,7 +102,7 @@ Components
    reference/debug
    reference/unit-testing
 
-In Depth Explanations / Further Reading
+Explicações detalhadas / Leituras Futuras
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (sergeyklay) [inline pt/index.rst]:
```diff
--- a/pt/index.rst
+++ b/pt/index.rst
 Explicações detalhadas / Leituras Futuras
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

**Comment by sergeyklay** [comment]
@MatheusRibak The reStructuredText requires strict adherence number of characters to emphasize headers

**Comment by MatheusRibak** [comment]
@sergeyklay  Yes, I did not pay attention to that, thank you.

File: pt/reference/forms.rst [hunk 4]
```diff
@@ -650,9 +645,9 @@ In addition to the form elements provided by Phalcon you can create your own cus
         }
     }
 
-Forms Manager
+Gerenciador de formulários
 -------------
```
Comment (sergeyklay) [inline pt/reference/forms.rst]: could you please update all headers?

**Comment by MatheusRibak** [comment]
@sergeyklay And now, Did I forget any?

**Comment by sergeyklay** [comment]
Thank you

**Pull request merged**